client/app/components/dialog-components-static/dialog-components-static.html

```diff
@@ -0,0 +1,23 @@
+<div class="container-fluid container-cards-pf">
+  <div class="row row-cards-pf">
+    <div class="col-xs-6 col-sm-4 col-md-4"
+         ng-repeat='dialogField in dialogComponentsStatic.components'
+         data-drag="true"
+         jqyoui-draggable="{animate:true, placeholder: 'keep', deepCopy: true}"
+         data-jqyoui-options="{revert: 'invalid', helper: 'clone'}"
+         ng-model="dialogField.placeholders">
+      <div class="card-pf card-pf-aggregate-status">
+        <h2 class="card-pf-title">
+          {{dialogField.label}}
+        </h2>
+        <div class="card-pf-body">
+          <p class="card-pf-aggregate-status-notifications">
+            <span class="card-pf-aggregate-status-notification">
+              <i class="fa {{dialogField.icon}}"></i>
```

Comment:
Maybe it'd be better to put the whole class list (including "fa") in `.icon`, in case one of the elements needs a pficon instead? (Aand `ng-class` is your friend ;).)